docs/bok/Activities/Level-3/Public-Development.md

```diff
@@ -8,32 +8,59 @@ sidebar_position: 1
 
+### Working with others
 
-something about using GitHub or GitLab.
+The biggest thing to remember about open source development is that it happens in the public sphere, in written text, with other humans. Text can be a challenging medium in which to interact with others. Tone and intent can be hard to communicate - emoji help. Language barriers often exist. For this reason it is important open source developers strive for clarity, compassion, and empathy in their communication.
 
-Risk of DLP. (we have articles already on this, we need to reference)
+Many folks have written in depth on this topic. The following articles dig a bit deeper into the human side of open, collaborative development:
 
-small PRs.
+* [Why the success of open source depends on empathy](https://opensource.com/article/21/2/open-source-empathy), by Bronagh Sorota
 
-## Pull Requests
+Developers' interactions, including code pushed and conversations that happen in Issues and Pull Requests are visible to the world.
 
-Submitting a PR/fixing a bug in an OSS project.
+### Publicly visible code
 
-QA as a process.
+A number of concerns come to mind when people think of publicly-visible code. Two of the most common may be summarized as:
 
-## Ways Of Contributing 
+* security risk
+* personal risk
 
-(from OSPO Mind Map)
+Security risk is most commonly the primary concern of the developer's employer. "What if private information is accidentally included in the code?" Thankfully, a number of tools and processes are available to mitigate this risk. These include:
 
-- Develop new features
-- Fix bugs
-- Improve docs
+* Data Loss Prevention (DLP) software - see this FINOS OSR [article on DLP](https://osr.finos.org/docs/bok/Artifacts/DLP-Software) for more information
+* Commit review software that requires approval before publication, including:
+  * [GitProxy](https://github.com/finos/git-proxy)
+  * [internal-contribution-forks](https://github.com/github-community-projects/internal-contribution-forks)
+
+Personal risk is related to any fear or concern an individual may have around participation in an open source community. Putting your work for the world can, at first, be a bit unnerving for some. Common concerns include not knowing what to do or doing the "wrong thing," receiving less than constructive feedback, or having their contribution rejected. 
+
+Healthy open source communities actively work to promote positive environments for contributors by:
+
+* Helping everyone understand the project's norms, standards, and processes
+* Setting and enforcing norms for communication
+* Pointing new contributors to small, easier to tackle opportunities
+
+GitHub has published a helpful blog of [tips for creating a healthy and sustainable open source community](https://github.blog/2022-07-07-healthy-and-sustainable-communities/). It is helpful for new contributors and maintainers alike to familiarize themselves with these practices and embody them in their open source work.
```

Comment:
Helpful is used in two sentences in a row. Should modify one of the two sentences to reduce duplication.

Reply:
Included, thank you 🙇‍♂️